extensions/adobe/experience/customerJourneyManagement/ajo-entity-mixins.schema.json

```diff
@@ -5,131 +5,178 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/experience/customerJourneyManagement/ajo",
+  "$id": "https://ns.adobe.com/xdm/ajo",
```

Comment:
Keep it consistent with the naming conventions for the customer journey management schemas. Use the same nomenclature as suggested for the class but with the field group name.

I would suggest to name this as something more meaningful and not just "ajo" as its too generic